### DISCUSSION TITLE
Disc numbers not showing as individual Discs

### DISCUSSION THREAD

**Discussion by ceefaxer** [croneter/PlexKodiConnect#1934] [Q&A]
I only use kodi for music and audiobooks. Most of my library is separated into albums with multiple disc numbers which are visible in plex. However these are not transferring over into kodi. When viewed in Estuary the first disc is always called 'Disc 0'. It contains all the albums tracks. The next disc is labelled 'Disc 2' and contains the expected number of tracks, disc 3 is ok etc. Once I use a skin (in my case arctic Horizon 2) discs are not shown at all. They are not separated into individual discs when I go into an album. It is just one long list of tracks grouped together in the correct order and track numbered but no reference to the disc number.

Installing a 'normal' kodi music library everything appears and is labelled correctly ie disc 1 (say 4 tracks) Disc 2 disc 3 etc. In a skin, the individual discs are shown with the correct amount of tracks.

I have tried Direct Paths with no change in the result. 

So it appears to be something to do with how it is handling the plex sync and disc numbers.

**Comment by bloqaudio** [comment]
I experience the same issue, and I think I fixed this, but I am not a developer in any way.  I came across this issue on a jellyfin github, https://github.com/jellyfin/jellyfin-kodi/issues/732

In PKC the file https://github.com/croneter/PlexKodiConnect/blob/python3-beta/resources/lib/itemtypes/music.py lines 444 to 448 seemed close enough to what was mentioned.  I commented out lines 445 to 447, back-indented line 448, repaired my library, multi disc and single disc albums are playing correctly.

My lines 444 to 448 now look like this.
`        disc = api.disc_number() or 1`
`        #if disc == 1:`
`        #    track = tracknumber`
`        #else:`
`        track = disc * 2 ** 16 + tracknumber`

Maybe create an issue from this so that it can be taken a look at properly?
